src/api/localStorage.js

```diff
@@ -0,0 +1,110 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+  }
+
+  createUser(user) {
+    this.updateUsers();
+    user.id = this.generateId();
+    this.users.push(user);
+    localStorage.setItem('UserAccounts', JSON.stringify(this.users));
+    console.log('User Added: ', user);
+    return this.users;
```

Comment:
This turns createUser into a createUserAndReturnUsers function. I'd just leave this return out and any time you need to get the users in your view component just make a call to getAllUsers.

Reply:
Complete